entrypoint.sh

```diff
@@ -105,7 +113,7 @@ if [ "${INPUT_STANDARD}" = "WordPress-VIP-Go" ] || [ "${INPUT_STANDARD}" = "Word
     git clone --depth 1 -b 2.3.3 https://github.com/Automattic/VIP-Coding-Standards.git ${HOME}/vipcs
     git clone https://github.com/sirbrillig/phpcs-variable-analysis ${HOME}/variable-analysis
 
-    decide_all_files_or_changed "${HOME}/wpcs,${HOME}/vipcs,${HOME}/variable-analysis"
+    decide_all_files_or_changed "${HOME}/vipcs,${HOME}/variable-analysis"
```

Comment:
```suggestion
    decide_all_files_or_changed "$(composer config home)/vendor/wp-coding-standards/wpcs,${HOME}/vipcs,${HOME}/variable-analysis"
```

This fixes the missing `WordPress.Security.EscapeOutput` mentioned at https://github.com/thrijith/autoshare-for-twitter/actions/runs/10938472447/job/30369473241?pr=2#step:4:76